packages/openapi-fetch/examples/react-query/src/lib/api/index.ts

```diff
@@ -1,5 +1,20 @@
-import createClient from "openapi-fetch";
+import createClient, { type Middleware } from "openapi-fetch";
 import type { paths } from "./v1";
 
+const throwOnError: Middleware = {
```

Comment:
People have [asked for this](https://github.com/drwpow/openapi-typescript/issues/1502): a more automatic way to interface with React Query. Hopefully this is sufficient!

Reply:
I feel this may be generally useful enough to deserve a call out in the middleware docs (even if just cross linking to the example). It may get missed otherwise